src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/callback/ExceptionRecordValidator.java

```diff
@@ -70,8 +71,15 @@ public ExceptionRecordValidator() {
         Validation<String, String> caseTypeIdValidation = hasCaseTypeId(caseDetails);
         Validation<String, String> poBoxValidation = hasPoBox(caseDetails);
         Validation<String, String> jurisdictionValidation = hasJurisdiction(caseDetails);
-        Validation<String, String> formTypeValidation = hasFormType(caseDetails);
         Validation<String, Classification> journeyClassificationValidation = hasJourneyClassification(caseDetails);
+
+        Validation<String, String> formTypeValidation;
+        if (journeyClassificationValidation.isValid() && journeyClassificationValidation.get().equals(EXCEPTION)) {
```

Comment:
Adding a comment would be great 

// Exception journey classification may not have form type so skipping validation for it